src/main/java/seedu/address/commons/events/model/PersonAddressDisplayDirectionsEvent.java

```diff
@@ -0,0 +1,23 @@
+package seedu.address.commons.events.model;
+
+import seedu.address.commons.events.BaseEvent;
+import seedu.address.model.person.Address;
+import seedu.address.model.person.ReadOnlyPerson;
+
+/** Indicates a person's address as a map is to be displayed*/
+public class PersonAddressDisplayDirectionsEvent extends BaseEvent {
+
+    public final ReadOnlyPerson person;
+    public final Address address;
+
+    public PersonAddressDisplayDirectionsEvent(ReadOnlyPerson person, Address address) {
+        this.person = person;
+        this.address = address;
+    }
+
+    @Override
+    public String toString() {
+        return "Displaying location of " + person.getName() + ": " + person.getAddress()
+                + " from " + address.toString();
```

Comment:
a bit unclear here...what does it mean by saying "person.getAddress() from address.toString()"? Does it mean it shows 'from the current address to the person's home address'?